src/uct/ib/base/ib_iface.c

```diff
@@ -205,6 +208,12 @@ ucs_config_field_t uct_ib_iface_config_table[] = {
    "Reverse Service level. 'auto' will set the same value of sl\n",
    ucs_offsetof(uct_ib_iface_config_t, reverse_sl), UCS_CONFIG_TYPE_ULUNITS},
 
+  {"PRIORITY_SLS", "0",
+   "List of SLs separated by comma, order determines which priority is using which SL\n"
+   "The later the SL is on the list the higher its priority",
```

Comment:
"Comma separated list of SLs to use for each priority level. The first SL on the list will be used for priority 0, second one for priority 1, etc."

BTW, how can we configure a certain priority level to use the default SL (the one specified by UCX_IB_SL or UCX_AR_ENABLE) ?

Reply:
I should support the 'auto' as value inside the array, but it seems UCS_CONFIG_TYPE_ARRAY won't support that

Reply:
Changing the array type from UCS_CONFIG_TYPE_UINT to UCS_CONFIG_TYPE_ULUNITS should enable auto as a value, auto would use ep->sl

I'll update the docs to mention setting auto as an option